src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_nodejs/nodejs_adexchangebuyer.v1.4.json.baseline

```diff
@@ -666,11 +614,54 @@ function authorize(callback) {
 // 1. If not already done, enable the Ad Exchange Buyer API
 //    and check the quota for your project at
 //    https://console.developers.google.com/apis/api/adexchangebuyer
-// 2. This sample uses Application Default Credentials for authentication.
-//    If not already done, install the gcloud CLI from
-//    https://cloud.google.com/sdk and run
-//    `gcloud beta auth application-default login`
-// 3. Install the Node.js client library by running
+// 2. Install the Node.js client library by running
+//    `npm install googleapis --save`
+
+var google = require('googleapis');
+var adexchangebuyer = google.adexchangebuyer('v1.4');
+
+authorize(function(authClient) {
+  var request = {
+    // The id for the account that will serve this creative.
+    accountId: 0,  // TODO: Update placeholder value.
```

Comment:
Is it easy to change so we get
```
accountId: '0'
```
in this case? This might relieve some confusion in Node.js.